tests/test_tztrout.py

```diff
@@ -0,0 +1,433 @@
+import datetime
+
+import pytest
+import tztrout
+from mock import patch
+
+
+class FakeDateTime(datetime.datetime):
+    "A datetime replacement that lets you set utcnow()"
+
+    @classmethod
+    def utcnow(cls, *args, **kwargs):
+        if hasattr(cls, 'dt'):
+            return cls.dt
+        raise NotImplementedError(
+            'use FakeDateTime.set_utcnow(datetime) first'
+        )
+
+    @classmethod
+    def set_utcnow(cls, dt):
+        cls.dt = dt
+
+
+class TestTZTrout:
+    def assert_only_one_tz(self, ids, tz_name, tz_names):
+        """ Assert that a given set of timezone ids only matches one tz name
+         in a given set of tz names
+         """
+        tz_names_copy = tz_names[:]
+        assert tz_name in tz_names_copy
+        tz_names_copy.remove(tz_name)
+        assert set(tztrout.tz_ids_for_tz_name(tz_name)) & set(ids)
+        for other_name in tz_names_copy:
+            assert not (set(tztrout.tz_ids_for_tz_name(other_name)) & set(ids))
+
+    @pytest.fixture
+    def us_ca_tz_names(self):
+        return ['PT', 'MT', 'CT', 'ET', 'AT']
+
+    @pytest.fixture
+    def au_tz_names(self):
+        return ['AWT', 'ACT', 'AET']
+
+    @pytest.mark.parametrize(
+        'phone, tz_ids',
+        [
+            ('+1 (650) 333 4444', ['America/Los_Angeles']),
+            ('+48 601 941 311', ['Europe/Warsaw']),
+        ],
+    )
+    def test_ids_for_phone(self, phone, tz_ids):
+        ids = tztrout.tz_ids_for_phone(phone)
+        assert ids == tz_ids
+
+    @pytest.mark.parametrize(
+        'country, state, city, zipcode, tz_ids, assume_any_tz',
```

Comment:
1. `tz_ids` -> `expected_tz_ids`.
1. I think `is_exact_match` would be a more easily understandable name vs `assume_any_tz`.

Reply:
Done in e2b8675